nibabel/freesurfer/io.py

```diff
@@ -44,24 +45,52 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_geometry(filepath):
+def _read_volume_info(fobj):
+    volume_info = OrderedDict()
+    head = np.fromfile(fobj, '>i4', 3)
+    if any(head != [2, 0, 20]):
+        warnings.warn("Unknown extension code.")
+    else:
+        volume_info['head'] = head
+        for key in ['valid', 'filename', 'volume', 'voxelsize', 'xras', 'yras',
+                    'zras', 'cras']:
+            pair = fobj.readline().decode('utf-8').split('=')
+            if pair[0].strip() != key or len(pair) != 2:
+                raise IOError('Error parsing volume info.')
+            volume_info[pair[0]] = pair[1]
```

Comment:
And is the consensus now to simply use strings? At least [at one point](https://github.com/nipy/nibabel/issues/456#issuecomment-223836410) the idea was that `volume_info['*ras']` would be numpy arrays.